messaging/messaging-core/src/main/java/uk/gov/justice/services/messaging/logging/ResponseLoggerHelper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.services.messaging.logging;
+
+import uk.gov.justice.services.common.http.HeaderConstants;
+
+import javax.json.Json;
+import javax.json.JsonObjectBuilder;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+import static uk.gov.justice.services.common.http.HeaderConstants.*;
```

Comment:
Replace wildcard with explicit imports - need to update IDE setttings to automate this.